modules/react/combobox/lib/ComboboxMenu.tsx

```diff
@@ -25,7 +26,7 @@ export const ComboboxMenu = createSubcomponent()({
      * of React. This means no extra props given to this component will be forwarded to the `div`
      * element, but the `ref` will be forwarded.
      */
-    Popper: Menu.Popper,
+    Popper: ComboboxMenuPopper,
```

Comment:
In case we want to change the popup behavior or style (changing the space between the menu and input for focus rings) I created a component to handle this change.